### PR TITLE
[4.0] Content Languages remove duplicate tip

### PR DESCRIPTION
**Pull request by brianteeman**
option=com_languages&view=languages had a title on the link and an identical tooltip (which was actually incorrect). This simple PR removes the tooltip

code review

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e254cdf619d5abf180a7c12d46ae4ec37d10ec7d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25730">issues.joomla.org/tracker/joomla-cms/25730</a>.</sub>

File: administrator/components/com_languages/tmpl/languages/default.php [hunk 1]
```diff
@@ -116,7 +116,6 @@
 									<?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'languages.', $canChange); ?>
 								</td>
 								<th scope="row">
-									<span class="editlinktip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', Text::_('JGLOBAL_EDIT_ITEM'), $item->title, 0); ?>">
```
Comment (wilsonge) [inline administrator/components/com_languages/tmpl/languages/default.php:119]: Something seems wrong here. You're deleting the span but not the close span. I'm also unsure how deleting the span effects the content in the span

Comment (richard67) [reply administrator/components/com_languages/tmpl/languages/default.php:119]: Oh, seems I missed that when testing?

**Comment by richard67** [comment]
I have tested this item :red_circle: unsuccessfully on e254cdf619d5abf180a7c12d46ae4ec37d10ec7d<br /><br />As George mentioned in his comment to the code, the closing span has not been removed with this PR together with the opening one, which results in invalid html.

Seems I missed that when testing the first time.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25730">issues.joomla.org/tracker/joomla-cms/25730</a>.</sub>

File: administrator/components/com_languages/tmpl/languages/default.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_languages/tmpl/languages/default.php:119]: I have removed the orphaned span - good spot

> . I'm also unsure how deleting the span effects the content in the span
It was incorrect code - hence removing it ;)

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 6bc99f1c209743d7fa314665da13f33bf9f4eb03<br /><br />Now it is ok. Removing the span does not result in any visible change.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25730">issues.joomla.org/tracker/joomla-cms/25730</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6bc99f1c209743d7fa314665da13f33bf9f4eb03<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25730">issues.joomla.org/tracker/joomla-cms/25730</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25730">issues.joomla.org/tracker/joomla-cms/25730</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**